src/renderers/react/createReactElementSpec.tsx

```diff
@@ -9,7 +9,7 @@ import { ElementProvider } from "./ElementProvider";
 
 export const createReactElementSpec = <FDesc extends FieldDescriptions<string>>(
   fieldDescriptions: FDesc,
-  consumer: Consumer<ReactElement, FDesc>,
+  consumer: Consumer<ReactElement | null, FDesc>,
```

Comment:
This matches the signature of the spec with React's definition of a React component, allowing us pass the TableForm component directly to 
 `createReactElementSpec`:

```ts
createReactElementSpec(tableFields, TableForm);

// rather than

createReactElementSpec(tableFields, props => <TableForm {...props} />);
```